lib/src/store/mod.rs

```diff
@@ -86,6 +88,18 @@ impl Storage {
         let imgstore = crate::imgstorage::Storage::create(&sysroot_dir, &self.run)?;
         Ok(self.imgstore.get_or_init(|| imgstore))
     }
+
+    /// Update the mtime on the storage root directory
+    #[context("Updating storage root mtime")]
+    pub(crate) fn update_mtime(&self) -> Result<()> {
+        let sysroot_dir = Dir::reopen_dir(&crate::utils::sysroot_fd(&self.sysroot))
```

Comment:
(We should have a `sysroot_dir` helper probably too)

Reply:
#980